sdk/documenttranslator/ai-document-translator-rest/review/ai-document-translator.api.md

```diff
@@ -4,738 +4,812 @@
-export interface GetDocumentStatus200Headers {
-    "retry-after"?: string;
+export interface DocumentTranslationGetDocumentStatus200Headers {
+    "retry-after"?: number;
     etag?: string;
 }
 
 // @public
-export interface GetDocumentStatus200Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus200Response extends HttpResponse {
     // (undocumented)
-    body: DocumentStatus;
+    body: DocumentStatusOutput;
     // (undocumented)
-    headers: RawHttpHeaders & GetDocumentStatus200Headers;
+    headers: RawHttpHeaders & DocumentTranslationGetDocumentStatus200Headers;
     // (undocumented)
     status: "200";
 }
 
 // @public
-export interface GetDocumentStatus401Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus401Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "401";
 }
 
 // @public
-export interface GetDocumentStatus404Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus404Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "404";
 }
 
 // @public
-export interface GetDocumentStatus429Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus429Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "429";
 }
 
 // @public
-export interface GetDocumentStatus500Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus500Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "500";
 }
 
 // @public
-export interface GetDocumentStatus503Response extends HttpResponse {
+export interface DocumentTranslationGetDocumentStatus503Response extends HttpResponse {
     // (undocumented)
-    body: TranslationErrorResponse;
+    body: TranslationErrorResponseOutput;
     // (undocumented)
     status: "503";
 }
 
 // @public (undocumented)
-export type GetDocumentStatusParameters = RequestParameters;
-
-// @public (undocumented)
-export interface GetSupportedDocumentFormats {
-    get(options?: GetSupportedDocumentFormatsParameters): Promise<GetSupportedDocumentFormats200Response | GetSupportedDocumentFormats429Response | GetSupportedDocumentFormats500Response | GetSupportedDocumentFormats503Response>;
-}
+export type DocumentTranslationGetDocumentStatusParameters = RequestParameters;
 
 // @public (undocumented)
-export interface GetSupportedDocumentFormats200Headers {
-    "retry-after"?: string;
+export interface DocumentTranslationGetSupportedDocumentFormats200Headers {
+    "retry-after"?: number;
```

Comment:
changing from string to number? could you find out what the typespec for this is?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/ebbdda9f9ba6cab2fbd4efccad59019a62649ea6/specification/cognitiveservices/data-plane/TranslatorText/stable/v1.0/TranslatorBatch.json#L764-L767